### PR TITLE
Homework1

### PR DESCRIPTION
**Pull request by Julie8899**
Can you help me to review my homework1? Thank you so much!

### CODE REVIEW

File: python/Homework.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    symbol="class",
+)
+fig.show()
+
+# From the scatter plot, Senoto has higher sepal width but lower sepal length compare to other two kinds
+# Scatter plot is easy to tell the difference if the difference is obvious among the kinds.
+
+# 3-2 Violin Plot
+
+df = px.data.tips()
```
Comment (huongphamBDA) [inline python/Homework.py]: I don't think we need the tips data here. Could you please check?

File: python/Homework.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+print(f"Classes: {random_forest.classes_}")
+print(f"Probability: {probability}")
+print(f"Predictions: {prediction}")
+
+# 4-4 Fit the features to a Decision Tree
+
+Decision_tree = tree.DecisionTreeClassifier(random_state=12)
+Decision_tree.fit(X, y)
+
+test_df = Iris_df.iloc[:, 0:4]
```
Comment (huongphamBDA) [inline python/Homework.py]: Line 169-174: You already defined these variables in section 4-3 above.

File: python/Homework.py [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+)
+Iris_df = pd.read_csv(
+    "https://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data",
+    names=heads,
+)
+Iris_df1 = pd.read_csv(
+    "https://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data"
+)
+
+nd_array_1 = np.array(Iris_df1)[:, 0:4]
```
Comment (huongphamBDA) [inline python/Homework.py]: slide [:, 0:4] could be replaced by [:, :4] for short

File: python/Homework.py [hunk 4]
```diff
@@ -0,0 +1,210 @@
+import numpy as np
+import pandas as pd
+import plotly.express as px
+from sklearn import tree
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+
```
Comment (huongphamBDA) [inline python/Homework.py:10]: Please use `def main()` here and `if __name__ == "__main__"...` at the end of your code.

Comment (Julie8899) [reply python/Homework.py:10]: I updated! Thank you for your comments!

File: python/Homework.py (hunk 3 above)
Comment (Julie8899) [reply python/Homework.py]: Thanks!

File: python/Homework.py (hunk 2 above)
Comment (Julie8899) [reply python/Homework.py]: Yes, I delete them! Thank you!

File: python/Homework.py (hunk 1 above)
Comment (Julie8899) [reply python/Homework.py]: No, we do not use it so I delete it!

Comment (dafrenchyman) [reply python/Homework.py]: Nice catch!

**Pull request merged**